packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -509,3 +510,54 @@ void sync_set_error_handler(realm_sync_config_t* sync_config, jobject error_hand
                                             get_env(true)->DeleteGlobalRef(static_cast<jobject>(userdata));
                                         });
 }
+
+void app_register_email_password(realm_app_t* app,
```

Comment:
We should merge this one after #517 to piggyback on the refactor of app callbacks: https://github.com/realm/realm-kotlin/pull/517/files#diff-b191a2b8280990335be6159a3281434b292df6d13e86ce0e05ff0c55085e1161R46